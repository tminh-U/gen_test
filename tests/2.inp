11
2 16 16 6 15 7 5 3 20 8 8 
10
5 9
5 11
11 11
3 8
3 6
4 9
2 7
9 10
11 11
9 11
