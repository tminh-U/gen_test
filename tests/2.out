10
22
2
9
7
16
10
3
2
5
